pkgs/development/libraries/hunspell/dictionaries.nix

```diff
@@ -242,7 +236,6 @@ let
 
       buildInputs = [ ispell perl hunspell ];
 
-      phases = ["unpackPhase" "installPhase"];
```

Comment:
Haven't tested it yet, but there is a Makefile in the sources, so now `buildPhase` would be run again, probably with unintended consequences.  `make` is called specifically in the install phase, so `buildPhase` probably needs to be skipped explicitly.

Reply:
i added `dontBuild = true;`